packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SurroundingPairScopeHandler/SurroundingPairInteriorScopeHandler.ts

```diff
@@ -32,28 +31,31 @@ export class SurroundingPairInteriorScopeHandler extends BaseScopeHandler {
     return this.surroundingPairScopeHandler.iterationScopeType;
   }
 
-  generateScopeCandidates(
+  *generateScopeCandidates(
     editor: TextEditor,
     position: Position,
     direction: Direction,
     hints: ScopeIteratorRequirements,
   ): Iterable<TargetScope> {
-    return map(
-      this.surroundingPairScopeHandler.generateScopes(
-        editor,
-        position,
-        direction,
-        hints,
-      ),
-      (scope) => ({
-        editor,
-        domain: scope.domain,
-        getTargets(isReversed) {
-          return scope
-            .getTargets(isReversed)
-            .flatMap((target) => target.getInterior()!);
-        },
-      }),
+    const scopes = this.surroundingPairScopeHandler.generateScopes(
+      editor,
+      position,
+      direction,
+      hints,
     );
+
+    for (const scope of scopes) {
+      if (!this.scopeType.requireSingleLine || scope.domain.isSingleLine) {
```

Comment:
This reads a touch easier to me.
```suggestion
      if (scope.domain.isSingleLine || !this.scopeType.requireSingleLine) {
```

Reply:
I think the intent is clearer the way I wrote it, but it's not to important either way.